### PR TITLE
fixed bug in triggering modal from card

### LINKED ISSUE #6: Create Carousel
- Create Carousel
- Should live in Home component
- What should go into Carousel?
- Most Likely Swiper Carousel install

### PR DESCRIPTION
**Pull request by DrSLMac**
# Pull Request

- [ ] New feature (adds functionality)
- [x] Describe the purpose of this PR(other): - fixed bug in triggering modal from anywhere on the card
- [x] I have added screenshots, when necessary (see below):
https://user-images.githubusercontent.com/102887963/190498591-8b81954d-3457-4baf-905f-6a38866bd3d1.mov

# Please describe the tests that you ran to verify your changes.

- [x] I tested my changes in the browser
- [ ] I tested with Cypress

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any other comments, questions or concerns?

### CODE REVIEW

File: components/BottomNavBar.js [hunk 1]
```diff
@@ -16,20 +16,19 @@ export default BottomNavBar;
 const styles = StyleSheet.create({
     bottomNavBar: {
         backgroundColor: '#39376E',
-        height: 90,
+        height: 105,
```
Comment (GOECHA) [inline components/BottomNavBar.js:19]: I like this adjustment- Nice refactor

File: components/CarouselCards/GoodVibeCard.js [hunk 2]
```diff
@@ -6,10 +6,10 @@ const GoodVibeCard = () => {
     return (
             <Pressable>
                 <View style={styles.card}>
-                    <ModalTemplate />
                     <Text style={styles.textStyle}>Tell Me Something Good</Text>
                     <Text style={styles.textStyle}>🪷</Text>
                     <Text style={styles.textStyle}>This is the carousel card that user can click to see the modal</Text>
+                    <ModalTemplate style={{zIndex:1}}/>
```
Comment (GOECHA) [inline components/CarouselCards/GoodVibeCard.js:12]: nice use of layering!

File: components/Header.js [hunk 3]
```diff
@@ -24,14 +24,14 @@ const styles = StyleSheet.create({
...
+        shadowColor: "#000",
+      shadowOffset: {
+        width: 6,
+        height: 6
+      },
+      shadowOpacity: 0.8,
+      shadowRadius: 10,
+      elevation: 5
         // ^^^ need to play with the shadow
         //thinking purple for the greeting and green for the header
```
Comment (GOECHA) [inline components/Header.js:36]: I agree, different shades would be nice soothing aesthetic

File: components/ModalTemplate.js [hunk 4]
```diff
@@ -69,18 +69,19 @@ const ModalTemplate = () => {
...
       height: 400,
       width: 300,
       borderRadius: 20,
       padding: 10,
       // elevation: 2,
       opacity: 0.0,
-      // zIndex: -1,
+      zIndex: 1,
     },
     // ^^ this styling will change with connection of modal to carousel
```
Comment (GOECHA) [inline components/ModalTemplate.js:81]: Got it, let's keep this comment in until the final refactor! 🤠

**Review by Maia-Cochran — APPROVED** [review]
I'm not sure I understand what the bug was here, but I'm glad it was refactored to fix! Can't wait to chat and see what you did here. Awesome speed on this breakthrough!

**Review by GOECHA — APPROVED** [review]
Keep up the momentum, making lots of headway today!!!

**Pull request merged**